python/src/aiconfig/schema.py

```diff
@@ -79,6 +79,15 @@ class OutputDataWithToolCallsValue(BaseModel):
     OutputDataWithToolCallsValue,
 ]
 
+class AttachmentDataWithStringValue(BaseModel):
+    """
+    This represents the input data that is storied as a string, but we use
+    both the `kind` field here and the `mime_type` to convert
+    the string into the input format we want.
+    """
+
+    kind: Literal["file_uri", "base64"]
+    value: str
```

Comment:
This type and code is pretty much a duplicate of 
```
class OutputDataWithStringValue(BaseModel):
    """
    This represents the output content that is storied as a string, but we use
    both the `kind` field here and the `mime_type` in ExecuteResult to convert
    the string into the output format we want.
    """

    kind: Literal["file_uri", "base64"]
    value: str
```

Should this be merged into one type and abstracted? That would reduce the explicitness and readability of this addition